### PR TITLE
Add HF.co for PRs / Issues regarding specific model checkpoints

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

Checkpoint issues should be put up directly to the Hub

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Perfect!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 blank_issues_enabled: true
 version: 2.1
 contact_links:
+  - name: Model checkpoints on the Hugging Face Hub
+    url: https://huggingface.co/models
+    about: Pull requests / Issue reports related to specific model checkpoints on the Hugging Face Hub
```
Comment (julien-c) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
    about: Open a Pull request / Discussion related to a specific model checkpoint directly on the Hugging Face Hub
```

**Review by julien-c — COMMENTED** [review]
Cool! we should also do this for `datasets` BTW

**Review by osanseviero — APPROVED** [review]
Very cool!! :fire:

**Review by sgugger — APPROVED** [review]
🔥🔥 🔥

**Comment by patrickvonplaten** [comment]
cc @lhoestq for datasets

**Pull request merged**